Jenkinsfile_CNP

```diff
@@ -6,12 +6,10 @@ def type = "java"
 def product = "rd"
 def component = "professional-api"
 
-List<LinkedHashMap<String, Object>> secrets = [
-    secret('s2s-url', 'S2S_URL'),
-    secret('s2s-secret', 'RD_P_S2S_SECRET'),
-    secret('s2s-microservice', 'RD_P_S2S_MICROSERVICE'),
-    secret('postgres-password', 'POSTGRES_PASSWORD'),
-    secret('postgres-username', 'POSTGRES_USERNAME'),
+def secrets = [
```

Comment:
Ok, Please document this change in the README and explain how it will effect the local build as per the new environment variables.